### PR TITLE
Fix part of #136: Topic overview low fi part 2 - Topic Download/Delete [ON HOLD]

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR is based on PR #233 and implements Topic Download and Delete functionality.

Mock: https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/b98da363-bac6-4f7d-b1d2-ee0f9aa048c4/Topic-w-o-Play-8

For testing purpose make following change in code:
1. Launcher activity should be `TopicActivity`
2. Inside `TopicActivityPresenter` change `TopicFragment` to `TopicOverviewFragment`

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is **assigned** to an appropriate reviewer.

### CODE REVIEW

**Comment by rt4914** [comment]
@veena14cs @nikitamarysolomanpvt This PR does not contain any test-cases but is ready in terms of functionality. You can have the first review.

File: app/src/main/res/layout/topic_download_dialog.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:padding="@dimen/cellular_data_dialog_padding">
```
Comment (veena14cs) [inline app/src/main/res/layout/topic_download_dialog.xml:6]: We can make use of FrameLayout if it contains only one view.

**Review by veena14cs — APPROVED** [review]
Looks good.

File: app/src/main/res/layout/topic_download_dialog.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout/topic_download_dialog.xml:6]: That would actually be the part of Hi-fi and I am currently working on low-fi only.

**Comment by rt4914** [comment]
@BenHenning This PR is ready for your review.

**Comment by BenHenning** [comment]
Thanks @rt4914. Apologies for the noise, but we're actually deprioritizing topic download/deletion flows for the prototype. Since we may not finish the infrastructure work for this (#169) in the prototype time frame, I'd rather not have the UI in place. Let's put this on hold for now (and keep the PR open) with the intention of completing it post prototype.

Let me know if you'd like to discuss this further.

**Comment by rt4914** [comment]
> Thanks @rt4914. Apologies for the noise, but we're actually deprioritizing topic download/deletion flows for the prototype. Since we may not finish the infrastructure work for this (#169) in the prototype time frame, I'd rather not have the UI in place. Let's put this on hold for now (and keep the PR open) with the intention of completing it post prototype.
> 
> Let me know if you'd like to discuss this further.

Okay no issues. This makes sense.

**Pull request closed without merging**